openupgrade_scripts/scripts/hr_holidays/15.0.1.5/pre-migration.py

```diff
@@ -0,0 +1,240 @@
+from openupgradelib import openupgrade
+
+
+def _fast_fill_hr_leave_employee_company_id(env):
```

Comment:
Odoo already fills with an SQL the related fields since https://github.com/odoo/odoo/pull/65232, so please remove all this method for not bloating the migration scripts.